docs/toc.md

```diff
@@ -189,6 +189,14 @@
 ### [dotnet-nuget-delete](core/preview3/tools/dotnet-nuget-delete.md)
 ### [dotnet-nuget-push](core/preview3/tools/dotnet-nuget-push.md)
 ### [dotnet-nuget-locals](core/preview3/tools/dotnet-nuget-locals.md)
+### Project modification commands
+#### References
+##### [dotnet-add-reference.md](core/preview3/tools/dotnet-add-reference.md)
```

Comment:
where is the "final desitination"? /preview4? 

There are a lot that need to move, wouldn't be easier to move them all when these are complete?